sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -263,6 +263,17 @@ object SQLConf {
     .booleanConf
     .createWithDefault(false)
 
+  val DISK_TO_MEMORY_SIZE_FACTOR = buildConf(
+    "spark.sql.sources.compressionFactor")
```

Comment:
merge this with the previous line

Reply:
BTW, how about `fileCompressionFactor`? Since it works for only file-based data sources.